src/Files.Uwp/Views/ModernShellPage.xaml.cs

```diff
@@ -848,6 +848,11 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
                 case (true, true, false, _, VirtualKey.Number7): // ctrl+shift+7, adaptive
                     InstanceViewModel.FolderSettings.ToggleLayoutModeAdaptive();
                     break;
+
+                case (true, false, false, true, VirtualKey.H): // ctrl + h, toggle hidden folder visibility
+                    var hiddenItemCurrentVisibitily = UserSettingsService.PreferencesSettingsService.AreHiddenItemsVisible;
```

Comment:
typo: `hiddenItemCurrentVisibility`

nit: can we move this closer to where the non-tab conditions are? somewhere before the `F1` shortcut should be sufficient